build.gradle

```diff
@@ -321,21 +321,21 @@ javadoc {
 
 		addBooleanOption "-allow-script-in-comments", true
 		links(
-				'https://guava.dev/releases/21.0/api/docs/',
-				'https://www.javadoc.io/doc/com.google.code.gson/gson/2.8.0/',
-				'https://logging.apache.org/log4j/2.x/log4j-api/apidocs/',
-				"https://javadoc.io/doc/org.jetbrains/annotations/${project.jetbrains_annotations_version}/",
-				'https://www.javadoc.io/doc/com.google.code.findbugs/jsr305/3.0.2/',
-				'https://javadoc.lwjgl.org/',
-				'https://fastutil.di.unimi.it/docs/',
-				'https://netty.io/4.1/api/',
-				'https://commons.apache.org/proper/commons-logging/javadocs/api-1.1.3/',
-				'https://commons.apache.org/proper/commons-lang/javadocs/api-3.5',
-				'https://commons.apache.org/proper/commons-io/javadocs/api-2.5',
-				'https://commons.apache.org/proper/commons-codec/archives/1.10/apidocs',
-				'https://javadoc.io/doc/org.apache.commons/commons-compress/1.21/',
-				"https://maven.fabricmc.net/docs/fabric-loader-${project.fabric_loader_version}/",
-				"https://docs.oracle.com/en/java/javase/17/docs/api/"
+				"https://guava.dev/releases/${project.guava}/api/docs/",
+				"https://www.javadoc.io/doc/com.google.code.gson/gson/${project.gson}/",
+				"https://logging.apache.org/log4j/2.x/log4j-api/apidocs/",
+				"https://javadoc.io/doc/org.jetbrains/annotations/${project.jetbrains_annotations}/",
+				"https://www.javadoc.io/doc/com.google.code.findbugs/jsr305/${project.findbugs}/",
+				"https://javadoc.lwjgl.org/",
+				"https://fastutil.di.unimi.it/docs/",
```

Comment:
Why were these all changed from single quotes to double quotes? Double quotes are only needed for ones with variables inside, otherwise literals should be single quotes

Reply:
Will also undo this then.